assets/src/edit-story/components/canvas/layout.js

```diff
@@ -19,7 +19,7 @@
  */
```

Comment:
Might be good to split this file up a bit in a future PR, it's growing quite big.

Reply:
Yeah, it's due for an overhaul for sure.